### DISCUSSION TITLE
Global GHG constraints and the resulted emission pathways

### DISCUSSION THREAD

**Discussion by Liyang-Guo** [JGCRI/gcam-core#321] [Q&A]
The paper "Ratcheting of climate pledges needed to limit peak global warming(https://doi.org/10.1038/s41558-022-01508-0)" constrained GHG emissions of regions in the world, and obtained the corresponding GHG emission pathway. The paper provides the source codes of GCAMv5.3_NDC version and the xml files of input policies(https://zenodo.org/record/7069066). 

I would like to use these xml files of input policies  and publically released GCAMv6 or v7 to replicate the GHG emission pathways in the paper. This allows me to avoid source code compiling that I cannot handle, and also to take advantage of the new features of recent GCAM. The attached zip file contains xml files for global regions' GHG emission constraints and policy link that I tried. 
[GHG constraint and policy link.zip](https://github.com/JGCRI/gcam-core/files/12243997/GHG.constraint.and.policy.link.zip)

The GHG emission pathways I get are a little weird. Table 1 lists the total constraints for GHG emissions, and emission pathway simulations from three versions of GCAM. Total GHG constraint in column (1) is the sum of emission constraints for the regions covered in the paper. GCAM5.3_NDC refers to the corresponding GHG emission pathway obtained from the mentioned paper. GCAM_v6 and GCAM_v7 in the table are the results simulated by corresponding public releases.
<img width="525" alt="Fig 1" src="https://github.com/JGCRI/gcam-core/assets/37605884/b2ed95b3-3b62-4d74-aaff-c1b33cbfbfef">

I have listed the results of GCAM_v6 and GCAM_v7 by emission source. The column (3) and (7) represent the total GHG emissions, which are obtained by adding the corresponding emission part and LUC part. The column (4) and (8) both come from two queries: CO2 emissions by region and nonCO2 emissions by region, emissions of latter are converted to CO2 equivalent via GWP. Column (5) and (9) both come from the query LUC emissions by region. For the GCAM_v6 and v7 results, I have some questions.

I. As the red font of column (3) and (7), GHG emissions since 2040 are greater than the corresponding value of total constraint and GCAM5.3_NDC in the paper, averagely 1Gt-2Gt CO2 equivalent. Are these results acceptable?  Or are these due to the relevant specifications of emissions in the different GCAM version?

II. As the blue font of column (3), (4), (7) and (8), GHG emissions in 2025 are smaller than those in 2020 and 2030, which is the weirdest part. This does not match the emission pathway obtained by the paper. And  I  checked the details of many regions both in GCAMv6 and v7, shown as Fig.1 and Fig.2 (see attached PDF files), their CO2 emissions are basically in similar situation, which does not match the trend of constraints given in the attached xml file (GHG constraint and policy link.zip).

III. As the highlighted font of column (5) and (9), CO2 emissions from LUC peaks in 2025, which is much larger than the emissions in 2020 and 2030. And for some regions both in GCAMv6 and v7, shown as Fig.1 and Fig.2 (see attached PDF files), their LUC CO2 emissions are basically in similar situation. From the data point of view, 2025 has produced huge emissions in land use change. Does this mean that there will be great change in land sector between 2021 and 2025?  And it seems that the CO2 emissions missing from the energy sector in 2025 will all be transferred to the LUC part, which also seems to cause the  situation in question II. What are the reasons for this result and can it be modified? 
[Fig.1 GCAMv6 CO2 emissions.pdf](https://github.com/JGCRI/gcam-core/files/12159399/Fig.1.GCAMv6.CO2.emissions.pdf)
[Fig.2 GCAMv7 CO2 emissions.pdf](https://github.com/JGCRI/gcam-core/files/12159401/Fig.2.GCAMv7.CO2.emissions.pdf)

When simulating with GCAM_v6 and v7, I did not change any of the default input files. I compared them with the input xml files of GCAM_v5.3_NDC version used by the paper, and there were indeed many differences in population, labor productivity, some income elasticities etc. Question II and III should be the ones that need to be addressed the most, but how should I adjust them?

References
Iyer, G., Ou, Y., Edmonds, J. et al. Ratcheting of climate pledges needed to limit peak global warming. Nat. Clim. Chang. 12, 1129–1135 (2022). https://doi.org/10.1038/s41558-022-01508-0
ou, yang. (2022). Model and input files for Iyer & Ou, et al. 2022 (Ratcheting of climate pledges needed to limit peak global warming ) [Data set]. Zenodo. https://doi.org/10.5281/zenodo.7069066

**Comment by Liyang-Guo** [comment]
I tried to change the income elasticity of industrial sectors such as iron_steel and aluminum from 2020 to 2025, and also tried to change the price-adjust of CO2_LUC in the policy link file, but the strange phenomenon of CO2 emission of LUC in 2025 in Figure 1 and Figure 2 did not change.

**Reply by daleihao** [reply]
Hi @Liyang-Guo, have you solved this issue?
